lib/parser.rb

```diff
@@ -9,5 +9,13 @@ def print_first_line
     first_line = file.readline
     file.close
     first_line
- end
+  end
+
+  def print_info
+    require 'json'
+    file = File.foreach(@file)
+    file_lines = file.count
+    json_lines = {@file => {'lines' => file_lines}}.to_json
```

Comment:
Voce esta transformando em json pra depois fazer o parse pra depois transformar em json novamente, da pra apenas passar o hash para o pretty_generate:

```suggestion
    JSON.pretty_generate({ @file => { lines: file_lines } }) 
```